docs/changelog.md

```diff
@@ -4,6 +4,11 @@ Lines for version numbers should always be formatted as `* MAJOR.MINOR.PATCH`
 with nothing else on the line.
 -->
 * HEAD
+* 0.18.0
+    * [feature] Add ability to limit the number of parallel runs within a pipeline
+    * [feature] Support for multiple bazel base container images
+    * [enhancement] Ensure at one list element is displayed and increase max number displayed
```

Comment:
I see all "enhancement"s have historically been written down as "feature"s.
We should probably codify this somehow.

Reply:
Enhancement is not like a new feature, just an improvement.